package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "nerds-oppose-society",
+  "version": "0.1.0",
+  "scripts": {
+    "prepare": "husky install"
```

Comment:
We could also have lint and/or test script in here which runs the script for both (ie does `cd frontend && npm test && cd ../backend && npm test`)

Reply:
Yep sure can! Would you also like `start-backend` & `start-frontend` (or `start:backend` & `start:frontend`). Similar thing for `build`?

Reply:
Similar concepts - they're probably more of a nice to have though. Should be pretty easy to do at any rate

Reply:
Cool I've implemented `test`, `lint`, `lint:fix` and `format` in c7db841